### DISCUSSION TITLE
SONOFF ZBMINI-L (home assistant and push switch)

### DISCUSSION THREAD

**Discussion by ThEKalYpsO** [Koenkk/zigbee2mqtt#21717] [Devices]
Good morning,

Does this object under home-assistant allow you to choose the type of switch: toggle or push.
According to the seller this is not possible under home-assistant, but only on other gateways: SonOff, Tuya...

Can you confirm this please?

Thanks,

**Comment by julianfs** [comment]
![image](https://github.com/Koenkk/zigbee2mqtt/assets/17122103/51d067c6-2055-48b7-93f3-5b4bb636ac39)

This is what I see on the 1 remaining I have in my networks. 

I wouldn't use them again, I'd go Shelly.
